part_2_dynamic.md

```diff
@@ -98,28 +127,26 @@ Other examples could be the
 * Calculate age of the participant
 * Try a dynamic image using some API or something...
 
-# Iterate over lists
+### Iterate over lists
 
 Imagine part of your pages looks like this:
 
 {% highlight html %}
 <ul>
-    <li>Painting</li>
-    <li>Eating</li>
-    <li>Cooking</li>
+    <li>French</li>
+    <li>English</li>
+    <li>Clingon</li>
     <li>Whatever</li>
 </ul>
 {% endhighlight %}
 
-Adding another hobby here means you have to repeat writing the HTML tags each and every time.
+Adding another language here means you have to repeat writing the HTML tags each and every time.
```

Comment:
locale instead of language to avoid potential confusion with programming language?

Reply:
We make the joke that one of the languages is Ruby and I actually don't mind that.